eclipse_development_process.adoc

```diff
@@ -310,182 +310,180 @@ digraph {
 
-It should be noted that the purpose of combining reviews is to better serve the community, rather than to reduce effort on the part of the project (though it is fortunate when it does both). Combining a progress and graduation review, or aggregating a progress review of a project and several of its child projects generally makes sense. Combining progress reviews for multiple unrelated projects most likely does not.
+It should be noted that the purpose of combining Reviews is to better serve the community, rather than to reduce effort on the part of the Project (though it is fortunate when it does both). Combining a progress and Graduation Review, or aggregating a Progress Review of a Project and several of its child Projects generally makes sense. Combining Progress Reviews for multiple unrelated Projects most likely does not.
 
 [#6_4_Releases]
 === 6.4 Releases
 
-Any project, with exception of permanent incubators, may make a release. A release may include the code from any subset of the project's descendants.
+Any Project, with exception of Permanent Incubators, may make a Release. A Release may include the code from any subset of the Project's descendants.
 
-_(Most of this section is paraphrased from the excellent http://www.apache.org/dev/release.html[Apache Software Foundation Releases FAQ]. The Eclipse community has many of the same beliefs about Releases as does the Apache community and their words were already excellent. The Apache Software Foundation Releases FAQ is distributed under the http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0].)_
+_(Most of this section is paraphrased from the excellent http://www.apache.org/dev/Release.html[Apache Software Foundation Releases FAQ]. The Eclipse community has many of the same beliefs about Releases as does the Apache community and their words were already excellent. The Apache Software Foundation Releases FAQ is distributed under the http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0].)_
 
-Releases are, by definition, anything that is distributed outside of the committers of a project. If consumers are being directed to download a build, then that build has been released (modulo the exceptions below). All projects and committers must obey the Eclipse Foundation requirements on approving any release.
+Releases are, by definition, anything that is distributed outside of the Committers of a Project. If consumers are being directed to download a build, then that build has been Released (modulo the exceptions below). All Projects and Committers must obey the Eclipse Foundation requirements on approving any Release.
```

Comment:
`Released` is also capitalized. Is this intentional?

Reply:
Not intentional. Nice catch.